app/screens/more_dms/more_dms.js

```diff
@@ -363,6 +364,8 @@ export default class MoreDirectMessages extends PureComponent {
         );
     };
 
+    filterUnknownUsers = (u) => Boolean(this.props.allProfiles[u.id])
```

Comment:
what does this do? do we need it for every  user or just for guests?

Reply:
maybe is only needed for guests, it handles the situation where you delete user profiles from the list of profiles that you have in the store while you are seeing this component. This only can happen if you are a guest.

I can apply this filter only for guests, I think is more efficient and is safe.

Reply:
Changed to only filter users this way if you are a guest